opentreemap/treemap/templates/treemap/partials/udf_edit_field.html

```diff
@@ -9,6 +9,7 @@
 </select>
 {% elif field.type == 'date' %}
 <input class="form-control" type="text" data-field-name="{{ field.name }}"
+       data-date-start-date="{{ datepicker_start_date|date:request.instance.short_date_format|datepicker_format }}"
```

Comment:
Good point. This is really a bandaid over a larger issue. We'll likely need to use a third party library to handle parsing and formatting dates prior to 1/1/1900.

The latest version of bootstrap datepicker has an option that would be very useful here called [assumeNearbyYear](https://bootstrap-datepicker.readthedocs.io/en/latest/options.html#assumenearbyyear). I could try to implement something similar, where, if the user entered a year that was interpreted as `00xx` we convert it to `20xx`?